DDD Workshop.ipynb

```diff
@@ -0,0 +1,398 @@
+    "\n",
+    "\n",
+    "#### Flight data\n",
+    "\n",
+    "In the following part, we will work with a dataset I have constructed specifically to make both high- and low-level decisions, as well as cleaning a dataset and detecting anomalies. This is not a real dataset, as will become clear, but hopefully, it is large enough for meaningful analysis and simple enough for interpretation with the real-world correlations.\n",
+    "\n",
+    "\n",
+    "\n",
+    "# Part 2: Knowledge into Decisions\n",
+    "\n",
+    "\n",
+    "#### Flight data, continued\n",
+    "\n",
+    "This dataset is made to represent flights leaving from SFO over a given time period, with info about the destination airport, the number of seats on the plane, a field called \"occupied\", and a time guage. We see the first few lines of the dataset here:\n",
+    "\n",
+    "`AIRPORT,SEATS,OCCUPIED,TIME`\n",
+    "`LAX,200,180,WEEKDAY`\n",
+    "`JFK,200,161,WEEKDAY`\n",
+    "`LAX,200,186,WEEKDAY`\n",
+    "`YVR,200,0.8315980689983522,WEEKDAY`\n",
+    "`LAX,250,227,REDEYE`\n",
+    "`JFK,250,226,REDEYE`\n",
+    "`LAX,250,228,REDEYE`\n",
+    "`YVR,250,0.8663797750002538,REDEYE`\n",
+    "&vellip;\n",
+    "\n",
+    "We will access the full 1800 line file in the code, but all of the contents will follow the format above.\n",
+    "\n",
+    "\n",
+    "## Types of Conclusions\n",
+    "\n",
+    "#### What makes a high-level conclusion?\n",
+    "\n",
+    "These types of conclusions are for very broad generalizations and are mostly observations, rather than predictions. There is a way to categorize the types of insights gained from data analysis: Descriptive, Diagnostic, Predictive, and Prescriptive. These can be thought of as increasing both specificities and work required: \n",
+    "\n",
+    "![](https://i.imgur.com/HlYuOwH.png)\n",
+    "\n",
+    "<!-- cite: https://www.scnsoft.com/blog/4-types-of-data-analytics -->\n",
+    "\n",
+    "A high-level conclusion is then one of the first two of these and is broader in scope. For example, in looking at the flight data we see that there are twice as many flights to LA relative to Vancouver or New York. This information is probably easy to see, and we can confirm it over the entire dataset:"
```

Comment:
Meaning descriptive and diagnostic? A little unclear

Reply:
good point, updated to backward/forward-looking